TestingLowerBounds/FDiv/Basic.lean

```diff
@@ -722,6 +750,14 @@ lemma fDiv_nonneg [IsProbabilityMeasure μ] [IsProbabilityMeasure ν]
   calc (0 : EReal) = f (μ Set.univ).toReal := by simp [hf_one]
   _ ≤ fDiv f μ ν := le_fDiv hf_cvx hf_cont
 
+lemma fDiv_nonneg_of_nonneg (hf : ∀ x, 0 ≤ f x) (hf_atTop : 0 ≤ derivAtTop f) :
```

Comment:
The hypothesis on derivAtTop is a consequence of hf and can be removed.
Also, the more general result we want is `fDiv_mono`, stating that fDiv is monotone in f. I guess we also want `fDiv_mono_ae`.

Reply:
Thanks, I have removed the hypothesis on `derivAtTop`.
Regarding `fDiv_mono` I wrote an attempted proof, but it relies on the monotonicity of `derivAtTop`, which is not true under the current definition; the underlying problem is that the `limsup` is not monotone over the reals, because `ℝ` is not a complete lattice. Maybe in the future we should change the definition of `derivAtTop`, putting the coercion to `EReal` inside the `limsup`, rather than outside of it, however for the moment I wrote some proofs with sorries, and comments documenting this.